lib/resources/r4/document_reference.yaml

```
@@ -2,13 +2,99 @@
 name: DocumentReference
 field_name_base_url: https://www.hl7.org/fhir/documentreference-definitions.html#DocumentReference
 fields:
+- name: resourceType
+  required: 'Yes'
+  type: string
+  description: The type of the FHIR resource.
+  example: |
+    {
+      "resourceType": "DocumentReference"
+    }
+  note: resourceType must be DocumentReference.
+  url: http://hl7.org/fhir/r4/documentreference.html
+
+- name: status
+  required: 'Yes'
+  type: code
+  action:
+    - create
+  description: The status of this document reference.
+  note: Only `current` is supported
+  url: http://hl7.org/fhir/r4/documentreference-definitions.html#DocumentReference.status
+  example: |
+    {
+      "status": "current"
+    }
+  binding:
+    description: The status of the document reference.
+    terminology:
+      - display: DocumentReference status
+        system: http://hl7.org/fhir/document-reference-status
+        info_link: https://www.hl7.org/fhir/valueset-document-reference-status.html
+
+- name: docStatus
+  required: 'Yes'
+  type: code
+  action:
+    - create
+  description: The status of the underlying document. Must be a supported Composition Status
+  note: Only `final` is supported for provider access. `final` and `amended` are supported for system access.
+  url: http://hl7.org/fhir/r4/documentreference-definitions.html#DocumentReference.docStatus
+  example: |
+    {
+      "docStatus": "final"
+    }
+  binding:
+    description: Status of the underlying document.
+    terminology:
+      - display: DocumentReference docStatus
+        system: http://hl7.org/fhir/r4/codesystem-composition-status.html
+        info_link: http://hl7.org/fhir/r4/valueset-composition-status.html
+
 - name: type
-  required: 'No'
+  required: 'Yes'
   type: CodeableConcept
-  description: Precise type of clinical document
+  action:
+    - create
+  description: Specifies the particular kind of document referenced.
+  note: The type must include a LOINC or a proprietary coding but not both together. Multiple LOINC codings or a single proprietary coding can be provided.
+        <br/><br/>
+        When providing proprietary code system, it should be of format 'https://fhir.cerner.com/&lt;your EHR source id&gt;/codeSet/&lt;code set&gt;' (where code set is a Millennium code set). Example&#58; 'https://fhir.cerner.com/eb2384f8-839e-4c6e-8b29-18e71db1a0b1/codeSet/72'.
```

Lets swap the tenant key to the new tenant key.

```suggestion
        When providing proprietary code system, it should be of format 'https://fhir.cerner.com/&lt;your EHR source id&gt;/codeSet/&lt;code set&gt;' (where code set is a Millennium code set). Example&#58; 'https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/72'.
```

---

You can find the changes here: https://github.com/cerner/fhir.cerner.com/pull/446/commits/1560e66bcece049dcfee1e781105538fe4d7898f